### ISSUE TITLE
Thousand Separator with dot {'.'} not display as it should be

### ISSUE THREAD

**Issue by irenefelita12** [s-yadav/react-number-format#34]
Hi,

I already installed this package and want to use it for currency input with custom thousand separator using dot (.)
For example I have a value 200000, I want to display it as 200.000
However, when I use the property thousandSeparator={'.'} it displayed as 2.00000

I also try on the demo from the codepen (http://codepen.io/s-yadav/pen/bpKNMa) 
and it is the same

Is it the correct behavior from this package?

Thanks

Regards,

Irene

**Comment by s-yadav** [comment]
Thats a bug. This is happening when you provide custom thousandSeparator as '.' but do not provide custom decimalSeparator (It takes . by default). 
I will fix this. For now give both props.
```js
<NumberFormat thousandSeparator={'.'} decimalSeparator={','} prefix={'$'} />
```

PS: codepen link had typo on decimalSeparator.  Updated that.

**Comment by irenefelita12** [comment]
Thanks.
Looking forward for the fix.

**Comment by marcel-k** [comment]
When will this be fixed?

The current version goes nuts when trying to combine for example
     ` thousandSeparator: '.',
        decimalSeparator: ',',
        decimalPrecision: 2,`

**Comment by s-yadav** [comment]
Fixed at 2.0.0 alpha
